src/components/AddControlKey.svelte

```diff
@@ -119,8 +119,8 @@
       classOverrides="border-silver border-2 rounded-lg"
     />
     <div class="flex w-350 justify-between">
-      <button on:click|preventDefault={addControlKey} class="btn-primary action-btn-l">Add It</button>
-      <button on:click|preventDefault={cancelAction} class="btn-cancel action-btn-r">Cancel Add </button>
+      <button on:click|preventDefault={addControlKey} class="btn-primary action-btn-l select-none">Add It</button>
```

Comment:
Instead of applying this to every button and then having to remember to do it for every future button, let's add it once in /style/app.css to <button />